src/app/src/util/hooks.js

```diff
@@ -12,7 +12,9 @@ import {
     maxVectorTileFacilitiesGridZoom,
 } from './constants.facilitiesMap';
 
-export default function useUpdateLeafletMapImperatively(
+import { CONFIRM_ACTION, MERGE_ACTION, REJECT_ACTION } from './constants';
+
+export function useUpdateLeafletMapImperatively(
```

Comment:
Could you please use arrow functions whenever possible? I aim to leverage the latest features of JavaScript.

Reply:
Done